src/main/java/org/killbill/billing/plugin/adyen/api/ExpiredPaymentPolicy.java

```diff
@@ -44,7 +46,7 @@ public boolean isExpired(final List<PaymentTransactionInfoPlugin> paymentTransac
             return false;
         }
 
-        final PaymentTransactionInfoPlugin transaction = latestTransaction(paymentTransactions);
+        final AdyenPaymentTransactionInfoPlugin transaction = (AdyenPaymentTransactionInfoPlugin) latestTransaction(paymentTransactions);
```

Comment:
Could we avoid the cast by changing `latestTransaction` to `public AdyenPaymentTransactionInfoPlugin latestTransaction(final List<AdyenPaymentTransactionInfoPlugin> paymentTransactions)`?